pkg/kubectl/cmd/delete.go

```diff
@@ -267,6 +267,7 @@ func ReapResult(r *resource.Result, f cmdutil.Factory, out io.Writer, isDefaultD
 		}
 		if waitForDeletion {
 			if err := waitForObjectDeletion(info, timeout); err != nil {
+				fmt.Fprintf(out, "Failed to wait for deletion: ")
```

Comment:
The prefix message should be still a part of the error message. We should not print to stdout.